project/api/models.py

```diff
@@ -139,7 +191,10 @@ class CachedData(models.Model):
 
     history = HistoricalRecords()
 
-    def save(self, *args, **kwargs):  # pylint: disable=W0221
+    def query_wikidata(self):
```

Comment:
This method is mocked by wiki_cd
https://github.com/chronhq/backend/blob/bf063e1b37d7dac37e9e2e2531ac5c7913538a67/project/api/tests/test_data.py#L36-L45